src/transformStateWithClones.js

```diff
@@ -7,7 +7,48 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArray = [];
+  let objectBox2 = {};
+
+  Object.assign(objectBox2, state);
+
+  for (let i = 0; i < actions.length; i++) {
+    const objectBox = {};
+
+    Object.assign(objectBox, objectBox2);
```

Comment:
you don't need these
make `objectBox2` copy of `state`, change `objectBox2` everywhere and push `objectBox2` to `resultArray`

variable `objectBox` you can remove